src/main/java/org/jboss/tools/intellij/openshift/actions/storage/CreateStorageAction.java

```diff
@@ -55,7 +53,6 @@ private CreateStorageDialog showDialog() {
   }
 
   public List<Integer> loadServicePorts(Odo odo, LazyMutableTreeNode projectNode, LazyMutableTreeNode applicationNode, LazyMutableTreeNode componentNode) {
```

Comment:
this method seems not used anymore. please remove it if so and the associated imports.